### PR TITLE
Add only the gold inside joke

### PR DESCRIPTION
**Pull request by Gabiras12**
- Add only the gold inside joke

### CODE REVIEW

File: source/inside-jokes.md [hunk 1]
```diff
@@ -38,3 +38,13 @@ And you have probably heard someone singing ["Beautiful"](https://www.youtube.co
 You have just opened Slack and found the weirdest gifs ever? Probably, it's wtfriday: you wake up, drink your coffe and contribute with the craziest gif you can find.
 
 ![Inside Jokes WTF](content/inside-jokes_wtfriday.gif)
+
+# Only the gold or _Só u ouro_
+
+If you find yourself asking: "what the hell _only the gold_ means?". Calm down. It's easy to catch.
+For most cultures, the gold color is a good thing. It's related with success, achievement and triumph.
```
Comment (renatocarvalho) [inline source/inside-jokes.md]: ...It's related `to` success...

File: source/inside-jokes.md [hunk 2]
```diff
@@ -38,3 +38,13 @@ And you have probably heard someone singing ["Beautiful"](https://www.youtube.co
...
+
+Gold in its physical state, by its very nature, denotes wealth and prestige in every country, culture and market in the world today - it is probably the most valuable and easily traded commodity available in the global market place.
```
Comment (renatocarvalho) [inline source/inside-jokes.md]: ...in the global `marketplace`...

File: source/inside-jokes.md [hunk 3]
```diff
@@ -38,3 +38,13 @@ And you have probably heard someone singing ["Beautiful"](https://www.youtube.co
...
+
+So, if you ask someone at the office: "Hey man, how're you?" and this person answer _only the gold_, it means that
```
Comment (renatocarvalho) [inline source/inside-jokes.md]: ...is in `well-being` state...

Comment (renatocarvalho) [inline source/inside-jokes.md]: Would change this:

So, if you ask someone at the office: `"Hey, how're you?"` and this person answer `"Only the gold!"`, it means that...

File: source/inside-jokes.md [hunk 4]
```diff
@@ -38,3 +38,13 @@ And you have probably heard someone singing ["Beautiful"](https://www.youtube.co
 You have just opened Slack and found the weirdest gifs ever? Probably, it's wtfriday: you wake up, drink your coffe and contribute with the craziest gif you can find.
 
 ![Inside Jokes WTF](content/inside-jokes_wtfriday.gif)
+
+# Only the gold or _Só u ouro_
```
Comment (renatocarvalho) [inline source/inside-jokes.md]: Será que deixa entre aspas?

"Only the gold" or "Só o ouro"

Comment (Gabiras12) [inline source/inside-jokes.md]: Coloquei aqui as aspas, acho q fica melhor sem.

**Pull request merged**